src/inspect_tool_support/src/inspect_tool_support/_remote_tools/_web_browser/accessibility_tree.py

```diff
@@ -1,10 +1,15 @@
 from functools import reduce
 from typing import Iterable, TypedDict
 
-from accessibility_tree_node import AccessibilityTreeNode
```

Comment:
I tried not to make any logic changes in the web browser tool's implementation. Most of these edits are simply updating import paths to support the new deployed package.